buildbot.mariadb.org/ci_build_images/common.Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN gosu buildbot curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs >/tm
     && rm -f /tmp/rustup-init.sh \
     # for Centos7/ppcle64, specific pip packages versions \
     # and python3-devel are needed \
-    && if grep -q "CentOS Linux release 7" /etc/centos-release && [ "$(arch)" = "ppc64le" ]; then \
+    && if [ grep -q "CentOS Linux release 7" /etc/centos-release || grep -q "Red Hat Enterprise Linux Server release 7.9 (Maipo)" /etc/redhat-release ]  && [ "$(arch)" = "ppc64le" ]; then \
```

Comment:
I suggest you use the following pattern, because if RedHat is upgraded to 7.10 it would not match:
`Red Hat Enterprise Linux Server release 7`

Reply:
Done